hibernate-reactive-core/src/test/java/org/hibernate/reactive/BaseReactiveTest.java

```diff
@@ -127,6 +130,26 @@ protected Configuration constructConfiguration() {
 		return configuration;
 	}
 
+	public CompletionStage<Void> deleteEntities(Class<?>... entities) {
+		return deleteEntities( Arrays.stream( entities )
+								  .map( BaseReactiveTest::defaultEntityName )
+								  .collect( Collectors.toList() )
+								  .toArray( new String[entities.length] ) );
+	}
+
+	private static String defaultEntityName(Class<?> aClass) {
+		int index = aClass.getName().lastIndexOf( '.' );
+		index = index > -1 ? index : 0;
+		return aClass.getName().substring( index );
```

Comment:
Complex Entity names that include a `.` will error due to the initial index for `.` is used as the start of the substring 
 and ends up with `.SomeName$Othername`